systemtests/bin/setup_upload_tests.py

```diff
@@ -70,57 +82,68 @@ def setup_upload_tests(max_size, start_page, auth_token, outputdir):
     )
     zip_path = os.path.join(outputdir, zip_filename)
 
-    click.echo(f"Generating ZIP file {zip_path} ...")
-    with tempfile.TemporaryDirectory(prefix="symbols") as tmpdirname:
-        sym_dir = os.path.join(tmpdirname, "syms")
-        tmp_zip_path = os.path.join(tmpdirname, zip_filename)
-
-        sym_files_url = urljoin(SYMBOLS_URL, "/api/uploads/files/")
-        sym_files_generator = get_sym_files(auth_token, sym_files_url, start_page)
-        for sym_filename, sym_size in sym_files_generator:
-            if sym_filename.endswith(".0"):
-                # Skip these because there aren't SYM files for them.
-                continue
-
-            is_try = False
-
-            if os.path.exists(tmp_zip_path):
-                # See if the new zip file is too big; if it is, we're done!
-                zip_size = os.stat(tmp_zip_path).st_size
-                click.echo(f"size: {zip_size:,}, max_size: {max_size:,}")
-                if zip_size > max_size:
-                    # Handle weird case where the first zip file we built was
-                    # too big--just use that.
-                    if not os.path.exists(zip_path):
-                        shutil.copy(tmp_zip_path, zip_path)
-                    break
-
-                # This zip file isn't too big, so copy it over.
-                shutil.copy(tmp_zip_path, zip_path)
-
-            click.echo(
-                click.style(f"Adding {sym_filename} ({sym_size:,}) ...", fg="yellow")
+    with METRICS.timer("elapsed_time"):
+        click.echo(f"Generating ZIP file {zip_path} ...")
+        with tempfile.TemporaryDirectory(prefix="symbols") as tmpdirname:
+            sym_dir = os.path.join(tmpdirname, "syms")
+            tmp_zip_path = os.path.join(tmpdirname, zip_filename)
+
+            params = {
+                "page": start_page,
+                "size": "< 10mb",
```

Comment:
Restricting the size reduces the likelihood we pick up a xul symbols file--they're huge and slow everything down.